py3status/formatter.py

```diff
@@ -339,6 +339,14 @@ def __init__(self, info):
             self.variable, self.value = info.split('<')
             self.condition = '<'
             self.check_valid = self._check_valid_condition
+        elif '!~' in info:
+            self.variable, self.value = info.split('!~')
+            self.condition = '!~'
+            self.check_valid = self._check_valid_condition
```

Comment:
`!~` is what I want to add too, but I don't know if that'll be accepted.

Without this, we have to use `'\?if=!long_str~long Oh my... What a long string.'` instead.

I sometimes used this by accident... `\?if!=number_of_updates` instead of `\?if=!number_of_updates`.